protocol/qrcode/connect_to_lao.json

```
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/jsonRPC.json",
+    "title": "QRCode data",
+    "description": "Match a data contained in a qrcode used to connect to an LAO.",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "server": {
+            "type": "string",
+            "format": "uri-reference",
+            "pattern": "(http|ws)s?://.*",
```

to ensure this pattern happens only at the beginning of the string:

```suggestion
            "pattern": "^(http|ws)s?://.*",
```